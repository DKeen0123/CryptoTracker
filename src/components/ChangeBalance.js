import React from 'react';

const ChangeBalance = props => {
  return (
    <div>
      <input className="input-balance" onChange={props.changeBalance} />
    </div>
  );
};

export default ChangeBalance;
