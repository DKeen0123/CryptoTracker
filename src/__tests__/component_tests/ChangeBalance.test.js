import React from 'react';
import { shallow } from 'enzyme';
import ChangeBalance from '../../components/ChangeBalance';

describe('ChangeBalance', () => {
  const changeBalance = shallow(<ChangeBalance />);

  it('renders correctly', () => {
    expect(changeBalance).toMatchSnapshot();
  });
});
