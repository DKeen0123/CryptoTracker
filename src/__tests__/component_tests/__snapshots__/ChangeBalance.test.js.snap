// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChangeBalance renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ChangeBalance
    changeBalance={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <input
        className="input-balance"
        onChange={[Function]}
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "input-balance",
        "onChange": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <input
          className="input-balance"
          onChange={[Function]}
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "input-balance",
          "onChange": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
